packages/mdc-data-table/README.md

```diff
@@ -173,3 +187,66 @@ Mixin | Description
 ## Accessibility
 
 Please refer [WAI-ARIA Authoring Practices for table](https://www.w3.org/TR/wai-aria-practices-1.1/#table) for ARIA recommended role, states & properties required for table element.
+
+## Events
+
+Please refer MDCDataTable's constant file to access these event constants.
```

Comment:
refer to

Reply:
This looks correct?

Reply:
Sorry - I meant: `Please refer MDC...` ---> `Please refer to MDC...`